config/webpack/webpack.config.js

```diff
@@ -133,6 +134,10 @@ const webpackConfig = (options, env, argv) => {
           'terra-navigation-prompt',
         ],
       }),
+      new webpack.DefinePlugin({
+        PACKAGE_VERSION: JSON.stringify(process.env.npm_package_version),
+        WEBPACK_BUILD_TIMESTAMP: JSON.stringify(new Date(Date.now()).toISOString()),
+      }),
     ],
```

Comment:
If you're wondering what's up with the `stringify`s, from the [docs](https://webpack.js.org/plugins/define-plugin/):

> Note that because the plugin does a direct text replacement, the value given to it must include actual quotes inside of the string itself. Typically, this is done either with alternate quotes, such as '"production"', or by using JSON.stringify('production').